intro-Linux/ch_beyond_the_basics.tex

```diff
@@ -477,24 +477,24 @@ \subsection{Example job script}
  #PBS -q long
  #PBS -m abe
  #PBS -j oe
- export HOMEDIR=$VSC_SCRATCH_VO_USER
+ export FILEDIR=$VSC_SCRATCH
```

Comment:
@redfast00 Sorry, I misunderstood this, I was mixing up with `WORKDIR`...

I'd rename this to `DATADIR`, and use `$VSC_DATA/example` as a value